politeiawww/invoices.go

```diff
@@ -540,6 +540,18 @@ func (p *politeiawww) validateInvoice(ni cms.NewInvoice, u *user.User) error {
 				}
 			}
 
+			// Check payment history for address
+			currentPayment, err := p.cmsDB.PaymentsByAddress(invInput.PaymentAddress)
+			if err != nil {
```

Comment:
I'm not able to submit an invoice because `PaymentsByAddress` is throwing a `invoice not found` error every time.  I think this may be setup incorrectly.

Reply:
This should be fixed for you now.